packages/ui/src/composables/useKeyboardFocusStyle.ts

```diff
@@ -0,0 +1,13 @@
+import { computed, ExtractPropTypes } from 'vue'
+
+import { useKeyboardOnlyFocus } from './'
+
+export const useKeyboardFocusStyleProps = { disableFocusStyle: { type: Boolean, default: false } }
+
+export function useKeyboardFocusStyle (props: ExtractPropTypes<typeof useKeyboardFocusStyleProps>) {
```

Comment:
Can't see any focus style here.

```suggestion
export function useKeyboardFocusable (props: ExtractPropTypes<typeof useKeyboardFocusStyleProps>) {
```